src/main/java/seedu/bigpp/BigPP.java

```diff
@@ -27,15 +35,30 @@ public void run() {
      * Runs the program until the user issues the exit command.
      */
     private void runLoopUntilExit() {
+        
+        LOGGER.setLevel(Level.INFO);
+        try {
+            FileHandler handler = new FileHandler("BigPP.log", true);
+            handler.setFormatter(new SimpleFormatter());
+            LOGGER.setUseParentHandlers(false);
+            LOGGER.addHandler(handler);
+            
+        } catch (IOException e) {
+            LOGGER.log(Level.SEVERE, "File logger not working.", e);
```

Comment:
can you actually do this? If logger fails you still log it will it actually be logged?

Reply:
no clue

Reply:
yes, logs to console instead of file